src/check/package-json.test.js

```diff
@@ -28,7 +25,7 @@ tap.ok(
   'package.json has "author" property'
 );
 
-tap.equal(pkg.main, 'es5/plugin.js', 'package.json "main" is "es5/plugin.js"');
+tap.equal(pkg.main, 'dist/es5/index.js', 'package.json "main" is "dist/es5/index.js"');
```

Comment:
- `jsnext:main` - Adding.
- `generator-videojs-plugin.version` - Adding.
- `files` - Adding.
- The rest is either related to optional features or relies on the `.yo-rc` which I want to avoid. But I will add verification of the spellbook configuration.